.github/workflows/reusable-revised-its.yml

```diff
@@ -118,6 +120,12 @@ jobs:
           echo $DRUID_IT_IMAGE_NAME
           docker save "$DRUID_IT_IMAGE_NAME" | gzip > druid-container-jdk${{ inputs.build_jdk }}.tar.gz
 
+      - name: Stop and remove docker containers
+        run: |
+          echo "Force stopping all containers and pruning"
+          docker ps -aq | xargs -r docker rm -f
```

Comment:
I am wondering if we should stop all containers. Instead, we can add labels to `services` in the docker-compose files and then remove the containers with those labels instead.